server/remote_cache/scorecard/scorecard.go

```diff
@@ -257,6 +266,53 @@ func blobName(invocationID string) string {
 	return filepath.Join(invocationID, blobFileName)
 }
 
+// ExtractFiles extracts files from invocation BES events, returning a mapping
+// of digest to file.
```

Comment:
nit: could you clarify which files are extracted?

Reply:
Done